js_task1/secondTask.js

```diff
@@ -0,0 +1,120 @@
+const generateButton = document.querySelector(".generate-btn");
+const checkButton = document.querySelector(".check-btn");
+
+const elementsCount = 6; // if we need moore domino, or less
+const minValue = 1;
+const maxValue = 3;
+
+let firstArray = [];
+let secondArray = [];
```

Comment:
Please, try to avoid such naming as 'first', 'second'.

Reply:
renamed